ansible/roles/stack-monitor-stateful/templates/alertrules.es.yml

```diff
@@ -3,13 +3,35 @@ groups:
   rules:
   - record: elasticsearch_filesystem_data_used_percent
     expr: 100 * (elasticsearch_filesystem_data_size_bytes - elasticsearch_filesystem_data_free_bytes) / elasticsearch_filesystem_data_size_bytes
-  - record: elasticsearch_filesystem_data_free_percent
-    expr: 100 - elasticsearch_filesystem_data_used_percent
+  - alert: elasticsearch_filesystem_data_free_percent_WARNING
+    expr: 100 - elasticsearch_filesystem_data_used_percent >= 30
+    for: 1m
+    labels:
+      severity: WARNING
+    annotations:
+      description: Elasticsearch Free space on Disk {% raw %}{{$value}}{% endraw %}
+      summary: Elasticsearch has less free disk space
+  - alert: elasticsearch_filesystem_data_free_percent_CRITICAL
+    expr: 100 - elasticsearch_filesystem_data_used_percent >= 20
```

Comment:
variablize all the limits and add them in defaults.yml of the role

Reply:
Resolved